include/psa/crypto_values.h

```diff
@@ -395,6 +395,7 @@
 /** Key for a cipher, AEAD or MAC algorithm based on the
  * Camellia block cipher. */
 #define PSA_KEY_TYPE_CAMELLIA                       ((psa_key_type_t)0x2403)
+#define PSA_KEY_TYPE_ARIA                           ((psa_key_type_t)0x2402)
```

Comment:
Please don't declare ARIA in this pull request. It's unrelated. The new value isn't even acceptable: all key types must have even parity (this allows an implementation to defend against single-bit fault injection).